src/storage/mvcc/reader/point_getter.rs

```diff
@@ -207,15 +222,18 @@ impl<S: Snapshot> PointGetter<S> {
                 self.met_newer_ts_data = NewerTsCheckState::Met;
             }
             if let Err(e) =
-                Lock::check_ts_conflict(Cow::Owned(lock), user_key, self.ts, &self.bypass_locks)
+                Lock::check_ts_conflict(Cow::Borrowed(&lock), user_key, self.ts, &self.bypass_locks)
             {
                 self.statistics.lock.processed_keys += 1;
+                if self.access_locks.contains(lock.ts) {
+                    return Ok(Some(lock));
```

Comment:
Should L227 be moved under this check as in the original logic `processed_keys` is increased in the error branch?

Reply:
I intend to do it because this lock is indeed processed.

Reply:
Make sense, the meaning of the `processed_keys`  changed a few times since release-4.0, I'm a bit confused about it.